.ci-pipelines/ci-manual-cleanup-script.sh

```diff
@@ -18,7 +18,10 @@ for this_test in $all_tests; do
     echo ""
     echo ">>>>>>>> Making clean for $this_test <<<<<<<<"
     echo ""
-    make -f Makefile_$this_test clean
+    make -f Makefile_$this_test distclean
```

Comment:
I realized that we need to make distclean instead of making clean when attempting to clean out any manually-run CI tests.  Otherwise it will not remove any of the KPP-generated code.